vertical-pod-autoscaler/pkg/recommender/checkpoint/checkpoint_writer_test.go

```diff
@@ -40,6 +40,7 @@ var (
 		model.ResourceCPU:    model.CPUAmountFromCores(3.14),
 		model.ResourceMemory: model.MemoryAmountFromBytes(3.14e9),
 	}
+	recommenderName = "default"
```

Comment:
Seem we have a few places need this default name. (cluster_feeder_test.go#60) can we have a const defined somewhere and all tests can share same reference.

Reply:
> Seem we have a few places need this default name. (cluster_feeder_test.go#60) can we have a const defined somewhere and all tests can share same reference.

@Jeffwan , do you have a suggestion on where to define it?
I can only think of adding a new go file in utils to define the name. Do you prefer to add a new file just for this constant or do you prefer the current implementation?

Reply:
This is minor. Current implementation is fine. Whenever we have more to support, we can refactor it later